File: app/src/androidTest/java/me/blog/korn123/commons/utils/FontUtilsTest.java
package me.blog.korn123.commons.utils;

import android.content.Context;
import android.support.test.InstrumentationRegistry;

import org.junit.BeforeClass;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

/**
 * Created by Administrator on 2017-11-06.
 */

public class FontUtilsTest {

    private static Context sContext;

    @BeforeClass
    public static void init() {
        sContext = InstrumentationRegistry.getTargetContext();
    }

    @Test
    public void test_03() {
        assertEquals("NanumPen", FontUtils.fontFileNameToDisplayName(sContext, "NanumPen.ttf"));
    }

    @Test
    public void test_04() {
        assertEquals("consola", FontUtils.fontFileNameToDisplayName(sContext, "consola.ttf"));
    }

}

File: app/src/androidTest/java/me/blog/korn123/commons/utils/FontUtilsTest.kt
package me.blog.korn123.commons.utils

import androidx.test.platform.app.InstrumentationRegistry
import org.junit.Assert.assertEquals
import org.junit.Test

/**
 * Created by Administrator on 2017-11-06.
 */

class FontUtilsTest {
    @Test
    fun test_03() {
        assertEquals("NanumPen", FontUtils.fontFileNameToDisplayName(InstrumentationRegistry.getInstrumentation().targetContext, "NanumPen.ttf"))
    }

    @Test
    fun test_04() {
        assertEquals("consola", FontUtils.fontFileNameToDisplayName(InstrumentationRegistry.getInstrumentation().targetContext, "consola.ttf"))
    }
}
